private/model/api/operation.go

```diff
@@ -269,7 +269,7 @@ func (e *example) traverseAny(s *Shape, required, payload bool) string {
 		str = e.traverseStruct(s, required, payload)
 	case "list":
 		str = e.traverseList(s, required, payload)
-	case "map":
+	case "map", "jsonvalue":
 		str = e.traverseMap(s, required, payload)
```

Comment:
For this, we don't even need to do that. We can just return a string `map[string]interface{}` and whether or not that is required. This is only used in example code.